tests/acceptance/SubscriberCountShortcodeCest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+use MailPoet\Test\DataFactories\Subscriber;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../DataFactories/Subscriber.php';
+
+class SubscriberCountShortcodeCest {
+  function createSubscriberCountPageWithShortcode(\AcceptanceTester $I) {
+    $I->wantTo('Create page with MP subscriber shortcode');
+    $segment_factory = new Segment();
+    $segment = $segment_factory->withName('SubscriberCount')->create();
+    $subscriber_factory = new Subscriber();
+    $subscriber_factory->withSegments([$segment])->create();
+    $pageTitle='OurSubscribers';
+    $pageContent='[mailpoet_subscribers_count\ segments="' . $segment->id . '"]';
```

Comment:
The same comment about `=`, but why do we need `\` here? and maybe you should consider using `sprintf` instead of concatenating strings. I think it looks a tad better and is easier to understand.

Reply:
Switching to using inline works, and it's even cleaner than sprintf, def easier to read than the concatenation. I avoided escaping the spaces with '\' with escapeshellarg(), per Rosta's advice on a different test. It's much more readable now, so thanks for putting me on the right path.

Reply:
I am very nervous about using inline string variables. They can be very dangerous. Because if you make a typo in the variable nothing stops you, and you don't even get a warning. Here is a nice article about that: https://www.tomasvotruba.cz/blog/2018/10/11/hi-my-name-is-tom-conctat-vs-in-sprintf-vs-in-string-variables/

But I guess it is ok in the tests. I would be definitely opposed to using them in the production code.